File: app/src/main/java/com/pedromassango/androidroomdatabasesample/data/Phrase.java
package com.pedromassango.androidroomdatabasesample.data;


import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * Created by Pedro Massango on 8/7/18.
 *
 * Entity class to store in Room Database
 */
@Entity(tableName = "phrase_table")
public class Phrase {

    @PrimaryKey
    @NonNull
    private String mPhrase;

    public Phrase(String mPhrase){
        this.mPhrase = mPhrase;
    }

    public String getPhrase() {
        return mPhrase;
    }
}

File: app/src/main/java/com/pedromassango/androidroomdatabasesample/data/Phrase.kt
package com.pedromassango.androidroomdatabasesample.data


import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by Pedro Massango on 8/7/18.
 *
 * Entity class to store in Room Database
 */
@Entity(tableName = "phrase_table")
class Phrase(var phrase: String,
             @PrimaryKey(autoGenerate = true)
             val id: Int = 0)
